frontend/src/components/blog-feed.tsx

```diff
@@ -46,9 +49,11 @@ export default function BlogFeed() {
               : `Posts related to "${selectedCategory}"`}
           </h1>
           <div className="flex flex-col gap-4">
-            {posts.slice(0, 5).map((post) => (
-              <FeaturedPostCard post={post} />
-            ))}
+            {posts.length === 0
+              ? Array(5)
+                  .fill(0)
```

Comment:
Can we create a component that returns `Array(5)
                  .fill(0)
                  .map((_, index) => <FeaturedPostCardSkeleton key={index} />)` and reuse it everywhere